prisma/schema.prisma

```diff
@@ -1128,14 +1128,14 @@ model BillingCustomer {
 }
 
 model ApiKey {
-  id          String    @id @default(cuid())
-  name        String?
-  active      Boolean   @default(true)
-  keyId       String    @unique
-  hashedToken String    @unique // scrypt hashed token
-  createdAt   DateTime  @default(now())
-  updatedAt   DateTime  @updatedAt
-  lastUsed    DateTime?
+  id         String    @id @default(cuid())
+  name       String?
+  active     Boolean   @default(true)
+  partialKey String
+  hashedKey  String    @unique // scrypt hashed token
```

Comment:
renamed field

Reply:
How about `hashedSecret` instead. We will stick to `scrypt` to secure `secret` instead of `createHash`